k0789546/quiz1.py

```diff
@@ -0,0 +1,24 @@
+############################
+#
+# Please complete this quiz
+#
+
+# declare an int variable named total with the value of 0
+total = int(0)
+# declare an array named numbers with all numbers 0 - 20
+numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20]
```

Comment:
Thank you for pointing out array vs list.

In the future for something like this you'd generally want to use `range(0,21)` instead of typing your numbers out.